auth-api/src/auth_api/models/invitation.py

```diff
@@ -0,0 +1,76 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This model manages a Invitation item in the Auth Service."""
+
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+
+from .base_model import BaseModel
+from .invitation_membership import InvitationMembership
+from .invite_status import InvitationStatus
+
+
+class Invitation(BaseModel):  # pylint: disable=too-few-public-methods # Temporarily disable until methods defined
+    """Model for a Invitation record."""
+
+    __tablename__ = 'invitation'
+
+    id = Column(Integer, primary_key=True)
```

Comment:
Having the invitation id as `uuid` would be better, to avoid any guessing on Rest endpoints. Also I guess this would be part of the invitation link.

Reply:
Invitation id will never be exposed directly in the url. It will be in an encoded format.The url will be something like 
https://auth-web-post-dev.pathfinder.gov.bc.ca/auth/token/confirm?token=NDM.XYEwoA.02jx8LU9vUe_UqyLcXC12rykruk